druid/src/widget/image.rs

```diff
@@ -106,7 +154,14 @@ impl<T: Data> Widget<T> for Image {
     }
 }
 
-/// Stored Image data.
+/// Loaded image data.
+///
+/// By default, Druid does not parse image data.
+/// However, [enabling the `image` feature](../index.html#optional-features)
```

Comment:
Let's move the link definition to the end, like most other documentation does it.